docs/markdown/Users.md

```diff
@@ -53,6 +53,7 @@ topic](https://github.com/topics/meson).
  - [GameMode](https://github.com/FeralInteractive/gamemode), a daemon/lib combo for Linux that allows games to request a set of optimisations be temporarily applied to the host OS
  - [Geary](https://wiki.gnome.org/Apps/Geary), an email application built around conversations, for the GNOME 3 desktop
  - [GIMP](https://gitlab.gnome.org/GNOME/gimp), an image manipulation program (master branch)
+ - [git](https://git-scm.com/), needs no introduction
```

Comment:
```suggestion
 - [git](https://git-scm.com/), the ubiquitous version control system
```

if a more formal / professional description is desired?

Reply:
We can also Go with Linus' original variant: "The information manager from hell" ;) Still exists in a couple of copyright notices. In any case, I don't mind much which of these it is, but one thing I need to insist on: Git is spelled with an upper-case letter, not with a lower-case one.

Reply:
Baha yes, I love it! 😂 
Good catch on capital G, thanks! ([and great song](https://www.youtube.com/watch?v=ynvgArZWToE))

I'll update my suggestion with both and commit it for @jpakkane's review